snuba/cli/multistorage_consumer.py

```diff
@@ -213,7 +213,11 @@ def multistorage_consumer(
             consumer_configuration, producer=producer, commit_log_topic=commit_log,
         )
 
-    metrics = MetricsWrapper(environment.metrics, "consumer")
+    metrics = MetricsWrapper(
+        environment.metrics,
+        "consumer",
+        tags={"group": consumer_group, "storage": storage_keys[0].value},
```

Comment:
Want to mark these metrics in a way so that it is recognizable that they are from a multistorage consumer, thus the storage we log is only one of the supported ones? 

Examples:
- another tag that identifies a multi storage consumer
- append `_m` to the storage name, or something like that ?

Reply:
Added _m to the storage name. Used that approach since most of the consumer graphs are based off storages.